### ISSUE TITLE
<autogenerated-issue>

### ISSUE THREAD

**Issue by hmanish79** [DeterminateSystems/nix-installer#1196]
## Error
```
Error: 
   0: Install failure
   1: Error executing action
   2: Action `create_nix_volume` errored
   3: Action `create_volume_service` errored
   4: Failed to execute command with status 5 `"launchctl" "bootout" "system" "/Library/LaunchDaemons/org.nixos.darwin-store.plist"`, stdout: 
      stderr: Boot-out failed: 5: Input/output error


```

## Metadata
|key|value|
|--|--|
|**version**|0.26.2|
|**os**|macos|
|**arch**|x86_64|

**Comment by cole-h** [comment]
Dupe of https://github.com/DeterminateSystems/nix-installer/issues/1187. Could you answer my questions at https://github.com/DeterminateSystems/nix-installer/issues/1187#issuecomment-2368473915 and report back in that issue, please?

**Comment by hmanish79** [comment]
To set some context on this issue, i had a prior installation of nix. I manually unmounted the `Nix Store` APFS volume using disk utility and rebooted the machine before running this installer. 

The issue seemed to be happening due to the lingering `/nix` folder. I had to remove the folder manually after another reboot as it was not letting me delete it. Once the folder was removed the installation completed smoothly.
